pkg/workspace/image/pusher.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/sh
+set -ex
+
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+TMPDIR="$(mktemp -d)"
+
+VOL_DIR="${1}"
+[ ! -z "${VOL_DIR}" ] || VOL_DIR='{{ .volDir }}'
+
+IMG_REF="${2}"
+[ ! -z "${IMG_REF}" ] || IMG_REF='{{ .imgRef }}'
+
+#{{`
+
+wait() {
+    until [ -e "${VOL_DIR}/fine_tuning_completed.txt" ]
+    do
+        sleep 1
+    done
+}
+
+mklayer() {
```

Comment:
construct an oci image from scratch. It's hard for somebody who are not familar with oci spec to maintain this script.
Can you add this document https://github.com/opencontainers/image-spec/blob/main/manifest.md#example-image-manifest as a comment.

Reply:
Yeah, sure!

Reply:
One extra thing to consider here, and unfortunately, there's no documentation for it, is that I'm building this image in a way that is architecture-independent and compatible with the K8s Image Volume implementations of `containerd` and `cri-o`. I got here by trial and error, tweaking stuff until both implementations accepted my images.

Reply:
I can write a README in this package, documenting my findings

Reply:
Done!